tox.ini

```diff
@@ -1,16 +1,11 @@
 [tox]
-envlist = py{27,36}, coverage, flake8
+envlist = py{27,36}, flake8
 
 [testenv]
-passenv = DISPLAY
-deps = -rrequirements/tests.txt
-commands = pytest --driver Firefox {posargs}
-
-[testenv:coverage]
 passenv = DISPLAY TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps = -rrequirements/coverage.txt
 commands =
-    pytest --driver Firefox --cov {posargs}
+    pytest --driver=Firefox --cov --html=results/{envname}.html {posargs}
```

Comment:
Does that mean we have coverage results for each supported Firefox release now and get x-amount of emails per push?